index.php

```diff
@@ -1,105 +1,68 @@
-                </button>
-            </div>
-        </div>
-    </form>
-    <div class="results"></div>
-</div>
-<div id="settings-dialog" class="modal fade" tabindex="-1" role="dialog">
-    <div class="modal-dialog" role="document">
-        <div class="modal-content">
-            <div class="modal-header">
-                <h5 class="modal-title">Settings</h5>
-                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                    <span aria-hidden="true">&times;</span>
-                </button>
-            </div>
-        <div class="modal-body">
-            <form id="settings">
-                <div class="form-group">
-                    <div class="custom-control custom-checkbox">
-                        <input type="checkbox" class="custom-control-input setting" name="extreme" id="extreme">
-                        <label class="custom-control-label" for="extreme">Show extreme games</label>
-                    </div>
-                    <div class="custom-control custom-checkbox">
-                        <input type="checkbox" class="custom-control-input setting" name="fpurl" id="fpurl">
-                        <label class="custom-control-label" for="fpurl">Show <em>flashpoint://</em> shortcuts</label>
-                    </div>
-                </div>
-            </form>
-        </div>
-        <div class="modal-footer">
-            <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
-        </div>
-    </div>
-</div>
-<script>
-var settings = JSON.parse(localStorage.getItem('settings')) || {};
-let search;
-$.fn.form = function() {
-    return Object.fromEntries(new FormData(this[0]));
-}
-function refresh() {
-    var data = {...$('#search').form(), ...settings};
-    search = $.post('search.php', data, function(data) {
-        $('.results').html(data);
-        $('.game-details').on('show.bs.collapse', function() {
-            search.abort();
-            var id = $(this).data('id');
-            var blur = !settings.extreme;
-            $(this).load(`view.php?id=${id}&blur=${blur}`, function() {
-                $(this).find('.blur').on('click', function() {
-                    $(this).removeClass('blur');
-                });
-            });
-        });
-    });
-}
-$('.setting').each(function() {
-    var val = settings[this.name];
-    if (this.type == 'checkbox') {
-        $(this).prop('checked', !!val);
-    } else {
-        $(this).val(val);
-    }
-});
-$('#settings-dialog').on('hidden.bs.modal', function() {
-    settings = $('#settings').form();
-    localStorage.setItem('settings', JSON.stringify(settings));
-    refresh();
-});
-$('#search').on('submit', function(e) {
-    e.preventDefault();
-});
-$('#search .form-control').on('input change', refresh);
-</script>
+	<head>
+		<meta charset="utf-8">
+		
+		<title>Flashpoint Browser</title>
+		
+		<script src="js/jquery-3.5.1.min.js"></script>
```

Comment:
Not convinced by the whole pulling the dependencies into the source tree, but ok.

Reply:
If you want to add dependencies, I would suggest including a `package.json` and then running `npm install` so that they're always local.

Reply:
Agreed, although it's out of scope for this PR.